docs/source/python/json.rst

```diff
@@ -38,6 +38,9 @@ The features currently offered are the following:
   such as ``my_data.json.gz``)
 * sophisticated type inference (see below)
 
+.. note::
+   Currently only the line-delimited JSON (JSON-LD) format is supported.
```

Comment:
I wouldn't use JSON-LD here (and the same in the docstring). 

My previous comment might have been worded a bit confusingly, but JSON-LD is the abbreviation for JSON-Linked Data, and searching for LDJSON ended up at pages about JSON-LD (which I raised as a argument to not use LDJSON). Given the variation in abbreviations used, maybe we can just leave out any abbreviation?

Reply:
That makes sense! I've removed these references to LD-JSON in the new commit